proof/refine/AARCH64/ArchAcc_R.thy

```diff
@@ -1038,6 +1051,17 @@ lemma getASIDPoolEntry_corres:
   apply simp
   done
 
+lemma no_0_page_table:
+  "\<lbrakk> no_0_obj' s; page_table_at' pt_t 0 s \<rbrakk> \<Longrightarrow> False"
```

Comment:
It depends on the use case which of these two makes more sense. The `.. ==> False` one is in normal form, so I'd usually prefer that, but if you want to use it for rewriting for instance, the `\<not> ..` form is more useful.

In terms of interpretation, the `False` form just says there is a contradiction between the two things and is usable symmetrically. The negation form assumes (in this case) a global property and says something local cannot be true because of it. Both make sense, depending on where you want to use them.